legion/engine/core/assets/functionalbinds.hpp

```diff
@@ -4,55 +4,63 @@
 namespace legion::core::assets
 {
     template<typename AssetType>
```

Comment:
Why are these no longer static?

Reply:
no idea tbh, they should be static... I'll change it back with the next PR